doc/client_lua_api.md

```diff
@@ -841,17 +841,25 @@ Please do not try to access the reference until the camera is initialized, other
 * `get_camera_mode()`
     * Returns with same syntax as above
 * `get_fov()`
+	* All values are in degrees
     * Returns:
 
 ```lua
      {
-         x = number,
-         y = number,
-         max = number,
-         actual = number
+         x = float,        -- Horizontal FOV (This is not a whole number as it is converted from radians)
+         y = float,        -- Vertical FOV (This is not a whole number as it is converted from radians)
+         max = float,      -- The maximum of x and y (This is not a whole number as it is converted from radians)
```

Comment:
"This is not a whole number" is not necessary because it is obvious from 'float'.
"converted from radians" is not necessary to know, that's a technical point.
Are these degrees or radians? "converted from radians" does not clearly mean 'degrees'.